ast.go

```diff
@@ -1320,7 +1320,7 @@ func (s *SelectStatement) RewriteFields(m FieldMapper) (*SelectStatement, error)
 
 				// Add additional types for certain functions.
 				switch call.Name {
-				case "count", "first", "last", "distinct", "elapsed", "mode", "sample":
+				case "count", "first", "last", "distinct", "elapsed", "mode", "sample", "count_distinct_approx":
```

Comment:
Would `count_approx` make more sense? `count_approx(distinct(v))`

Reply:
Maybe? My argument against is that `count_approx` is never useful except in front of `distinct`. Also consider: `approx(count(distinct(v)))`.